webrtc.html

```diff
@@ -1272,7 +1272,8 @@
                           </li>
                           <li>
                             <p><a>Create an RTCRtpTransceiver</a> with
-                            <var>sender</var> and <var>receiver</var>, and
+                            <var>sender</var>, <var>receiver</var> and
+                            "inactive" as direction, and
```

Comment:
Should be with "recvonly" direction; we want to offer to receive things by default. See JSEP 5.9:

>          +  If no RtpTransceiver was found in the previous step, create
>             one with a recvonly direction.